### PR TITLE
234 create more user app mixins

### PR DESCRIPTION
**Pull request by sakarisalmi**
Created a bunch of User app mixins.
Also formatted User app files to follow pep8 style.

### CODE REVIEW

**Comment by merituulie** [comment]
Did you test all of the functions through frontend in this PR?

**Comment by sakarisalmi** [comment]
> Did you test all of the functions through frontend in this PR?

I did, and all the User stuff works

File: backend/User/validators.py [hunk 1]
```diff
@@ -2,16 +2,32 @@
 import os
 
 def validate_image(file):
-	valid_mime_types = ['image/jpeg', 'image/png', 'image/gif']
-	valid_file_extensions = ['.jpg', '.jpeg', '.png', '.gif']
-	max_file_size = 2097152
+    """
```
Comment (merituulie) [inline backend/User/validators.py:5]: mmeh, as said before, summariesa are quite unnecessary and get easily obsolete if not kept up to date...

File: backend/User/mixins.py
Comment (merituulie) [inline backend/User/mixins.py:1]: I like this, picasso! Separating different database functionality to their own functions, very nice. This is not up to date though, fetch and pull from main! but Im not sure as we have a manager and a validator, what is the purpose of the mixin compared to those two... in anycase, dot really care at this point however meshed the responsibilities are but in general something to think about.

**Comment by merituulie** [comment]
This PR has some conflicts. Is it still under construction or can it be re-reviewed already?

**Review by azarSarikhani — APPROVED** [review]

**Comment by sakarisalmi** [comment]
Everything is now in order with this PR. I've tested all the stuff where the mixins are used and everything works like it should.

**Review by dorianjagusch — APPROVED** [review]

**Pull request merged**